cdap-ui/app/cdap/components/ProductsDrawer/index.js

```diff
@@ -21,25 +21,30 @@ import classnames from 'classnames';
 require('./ProductsDropdown.less');
 import head from 'lodash/head';
 import shortid from 'shortid';
+import NamespaceStore from 'services/NamespaceStore';
 
 export default class ProductsDrawer extends Component {
   constructor(props) {
     super(props);
+    this.namespace;
     let products = [
       {
-        link: '/cask-cdap',
+        baselink: '/cask-cdap/',
+        link: '/cask-cdap/',
         label: T.translate('commons.cdap'),
         name: 'cdap',
         icon: 'icon-fist'
       },
       {
-        link: '/cask-hydrator',
+        baselink: '/cask-hydrator/',
```

Comment:
having baselink here just for reference to construct link is not good. You should just update the link directly and remove baselink